Gems/Atom/Tools/MaterialEditor/Code/Source/MaterialEditorApplication.h

```diff
@@ -92,28 +73,14 @@ namespace MaterialEditor
         void AssetSystemAvailable() override;
         //////////////////////////////////////////////////////////////////////////
 
-        //////////////////////////////////////////////////////////////////////////
-        // AZ::UserSettingsOwnerRequestBus::Handler overrides...
-        void SaveSettings() override;
-        //////////////////////////////////////////////////////////////////////////
-
         //////////////////////////////////////////////////////////////////////////
         // AZ::Debug::TraceMessageBus::Handler overrides...
         bool OnOutput(const char* window, const char* message) override;
         ////////////////////////////////////////////////////////////////////////// 
 
-        void CompileCriticalAssets();
-
-        void ProcessCommandLine(const AZ::CommandLine& commandLine);
-
-        void LoadSettings();
-        void UnloadSettings();
-
-        bool LaunchDiscoveryService();
-
-        void StartInternal();
-
-        static void PyIdleWaitFrames(uint32_t frames);
+        void CompileCriticalAssets() override;
```

Comment:
If you pass in the list of assets and paths or supply them with a function or variable then this function only needs to be in the base class